src/types.ts

```diff
@@ -0,0 +1,108 @@
+interface Attribute {
```

Comment:
Most of these need to be exported it seems.

The scenarios and persona's for sure need to be exported like you are already doing. They depend on the other interfaces, thus they also need to be exported

Reply:
I'm exporting on-demand